src/providers/twitch/TwitchChannel.cpp

```diff
@@ -437,6 +439,51 @@ void TwitchChannel::setLive(bool newLiveStatus)
     }
 }
 
+void TwitchChannel::refreshTitle()
+{
+    auto roomID = this->roomId();
+    if (roomID.isEmpty())
+    {
+        return;
+    }
+
+    if (this->titleRefreshedTime_.elapsed() < 10 * 1000)
+    {
+        return;
+    }
+    this->titleRefreshedTime_.start();
```

Comment:
I would prefer assigning `QTime::currentTime()` to it since `start` is very undescriptive of what it does.